File: exam-preparation/subiecte-java/topics/src/main/java/ro/ubb/topics/model/Post.java
package ro.ubb.topics.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import net.bytebuddy.dynamic.loading.InjectionClassLoader;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Post {

    @Id
    @GeneratedValue
    private Long id;

    private String userName;

    private int topicId;

    private String text;

    private int date;
}

File: exam-preparation/subiecte-c#/Topics/Topics/Models/Post.cs
using System;
using System.Collections.Generic;

namespace Topics.Models
{
    public partial class Post
    {
        public int Id { get; set; }
        public string User { get; set; }
        public int? TopicId { get; set; }
        public string Text { get; set; }
        public int? Date { get; set; }

        public override string ToString()
        {
            return $"{nameof(Id)}: {Id}, {nameof(User)}: {User}, {nameof(TopicId)}: {TopicId}, {nameof(Text)}: {Text}, {nameof(Date)}: {Date}";
        }
    }
}
